### PR TITLE
Remove objexx gio part 2

### PR DESCRIPTION
**Pull request by lefticus**
Pull request overview
---------------------

AirLoopNetworkBalanceManager.cc
HeatBalanceManager.cc

have had their EIO output updated to use the new OutputFiles with libfmt.

* Documentation updates provided
* Proper rounding to match old Objexx RoundSigDigits
* 3 digit exponents provided to match Objexx RoundSigDigits

### CODE REVIEW

**Comment by lefticus** [comment]
@Myoldmopar I just kicked the Mac build to make it rebuild. I cannot fathom why it would have so many test failures when everything else is perfect

**Comment by Myoldmopar** [comment]
Yeah we had to restart the Mac this afternoon, it should be fine now.

**Comment by mjwitte** [comment]
@lefticus FYI, this branch won't build unit tests locally on Windows. Same problem as #7675 showing up in many unit test files on this branch.
[unittestbuildlog.txt](https://github.com/NREL/EnergyPlus/files/4059609/unittestbuildlog.txt)

**Comment by lefticus** [comment]
I'll pull from develop and make sure it's coming out clean after that.

**Comment by lefticus** [comment]
@mjwitte @Myoldmopar do either of you have any insight here? I had all tests passing with no diffs on Windows last night. After merging the fixes for Windows from develop I'm now getting failures only on Windows.

Is the Windows build box known to be misbehaving?

**Comment by Myoldmopar** [comment]
It wasn't known to be misbehaving, but every once in a while it does.  I wouldn't re-run CI at this point though because a branch merged into develop a few minutes ago that will then trigger lots of unrelated diffs here.  If you are completely done with this branch and ready for review, we'll review it based on the other platforms, test it locally, and if we need to we could pull develop in one more time just to confirm.

**Comment by Myoldmopar** [comment]
Oh, and @lefticus, do you need any other info from @mjwitte about the unit test issue?  Last time the refactor branch got in without us realizing that it wouldn't build locally on Windows in debug mode.

**Comment by lefticus** [comment]
I don't think I need anything else from @mjwitte I'd been relying on the dashboard to tell me if something was amiss, but I'll make sure I can run my own Windows builds as well from now on.

From my perspective this branch is done, and I can start working on the next set of outputs, trying to keep the merge diffs small.

**Comment by mjwitte** [comment]
@lefticus This branch is not ready to go. Units tests do not build in debug config on Windows. See https://github.com/NREL/EnergyPlus/pull/7677#issuecomment-574200062 The build output file there shows which files are failing. Even when CI is running normally, it doesn't do a debug config on the windows platform, so you won't see it without doing a local build. We need to consider adding a debug build on CI. @Myoldmopar

**Comment by lefticus** [comment]
@mjwitte clearly I misunderstood, I thought the point of your patches pushed to develop was to get these builds working on Windows Debug.

I'll fix it and get the PR retested

**Comment by mjwitte** [comment]
@lefticus Thanks. Sorry for the confusion. The patch just fixed gio part 1. Part 2 has now touched more unit test source files. I think all you need to do is delete the `using namespace ObjexxFCL;` line in the affected files. And match the include style formatting if need be. The other changes I made over there were extra cleanup just because I was in the file and saw that.

File: third_party/fmt-6.1.2/include/fmt/core.h [hunk 1]
```diff
@@ -233,7 +233,7 @@ FMT_API void assert_fail(const char* file, int line, const char* message);
 #    define FMT_ASSERT(condition, message) \
       ((condition)                         \
            ? void()                        \
-           : fmt::internal::assert_fail(__FILE__, __LINE__, (message)))
+           : ::fmt::internal::assert_fail(__FILE__, __LINE__, (message)))
```
Comment (mjwitte) [inline third_party/fmt-6.1.2/include/fmt/core.h:236]: @lefticus Thanks for the proper fix. Confirmed this branch now builds locally for debug config on windows.

**Comment by lefticus** [comment]
@Myoldmopar your call - do I update from develop again to deal with the new diffs?

**Comment by mjwitte** [comment]
@lefticus IMHO, yes, please, update from develop one more time.

**Comment by Myoldmopar** [comment]
Yeah, a pull from develop wouldn't hurt

**Comment by Myoldmopar** [comment]
This does look good to me.  I think I'll go ahead and drop this in as it only has the one tiny EIO diff so shouldn't interfere heavily with other builds.

**Comment by lefticus** [comment]
And that tiny EIO diff was necessary to produce further ones. So great!

**Pull request merged**